src/lib/containers/TermsAndConditions.tsx

```diff
@@ -0,0 +1,211 @@
+    sigPadRef?.clear()
+    setTrimmedDataUrl(undefined)
+    resetSignatureDialog(true)
+  }
+  const closeSignatureDialog = () => {
+    setTrimmedDataUrl(undefined)
+    resetSignatureDialog()
+  }
+  const submitSignature = () => {
+    if (showSignaturePad && !sigPadRef?.isEmpty()) {
+      setTrimmedDataUrl(sigPadRef?.getTrimmedCanvas().toDataURL('image/png'))
+      resetSignatureDialog()
+    }
+  }
+  const resetSignatureDialog = (showDialog: boolean = false) => {
+    setShowDialog(showDialog)
+    setShowSignaturePad(false)
+    setSubmitBtnActive(false)
+  }
+
+  return(
+    <section className="terms-conditions">
+      <h3 className="page-header">Synapse Terms and Conditions Agreement</h3>
+      <form>
+        <label>
+          I affirm my commitment to all Synapse Governance policies for responsible research and data handling, including:
+        </label>
+        <ul className="term-list">
+        { tcList.length &&
+            tcList.map((item, i) => {
+              return (
+                <li key={`tc-item-${i}`} className={checkboxEnabled[i] === true ? "terms-enabled" : ""}>
+                  <TermsAndConditionsItem
+                    item={item}
+                    id={i}
+                    checked={checkboxChecked[i]}
+                    enabled={checkboxEnabled[i]}
+                    onChange={updateCheckboxState}
+                  />
+                </li>
+              )
+        })}
+        </ul>
+        <div className="view-terms"><Button href={tcAgreement} target="_blank">View Complete Terms and Conditions for Use</Button></div>
+        <div className="terms-signature">
+          <div><label>Your Signature</label></div>
+          <Modal animation={false} show={showDialog} className="signature-pad-modal" onHide={closeSignatureDialog}>
+            <Modal.Header closeButton>
+              <Modal.Title>Sign Agreement</Modal.Title>
+            </Modal.Header>
+            <Modal.Body>
+              { !showSignaturePad &&
+                <label style={canvasDimension} onClick={displaySignaturePad} className="signature-pad-label">Tap to sign</label>
+              }
+              { showSignaturePad &&
+                <SignatureCanvas
+                  ref={(ref) => sigPadRef = ref}
+                  canvasProps={canvasDimension}
+                  onEnd={() => setSubmitBtnActive(true)}
+                />
+              }
+            </Modal.Body>
+            <Modal.Footer>
+              <Button variant="secondary" onClick={clearSignature}><RefreshSvg />CLEAR SIGNATURE</Button>
+              <Button variant="secondary" onClick={closeSignatureDialog}>CANCEL</Button>
+              <Button
+                variant="primary"
+                className={submitBtnActive ? "" : "btn-disabled"}
+                active={submitBtnActive}
+                onClick={submitSignature}>Submit Signature
+              </Button>
+            </Modal.Footer>
+          </Modal>
```

Comment:
FYI, we may not need the signature pad according to Stockard and she is checking with Governance. But I implemented it anyway just in case.